1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,241 @@
+---
+post_title: Quick Start
+menu_order: 3.3
+---
+
+Use this guide to get started with the `dcos exec` debugging command.
+
+**Prerequisite:**
+
+- A container launched by using the [DC/OS Universal container runtime](/docs/1.9/usage/containerizers/).
+
+
+# Launch a long running interactive Bash session
+
+In this example, a long running job app is launched and then the `dcos exec` command is used to launch an interactive Bash session inside of the task container on the node.
+
+1.  Deploy and run a job app with the DC/OS CLI:
+
+    1.  Create the following app definition and save as `⁠⁠⁠⁠test-job.json`. This specifies a sleep job that runs for `10000000` seconds.
+    
+        ```bash
+        {
+          "id": "task-exec-test",
+          "labels": {},
+          "run": {
+            "artifacts": [],
+            "cmd": "sleep 100000000",
+            "cpus": 0.01,
+            "disk": 0,
+            "env": {},
+            "maxLaunchDelay": 3600,
+            "mem": 32,
+            "placement": {
+              "constraints": []
+            },
+            "restart": {
+              "policy": "NEVER"
+            },
+            "volumes": []
+          }
+        }
+        ```
```

Comment:
Did something accidentally get reverted here? I think the job definition should just be:
```
{
  "id": "my-job",
  "run": {
    "cpus": 0.01,
    "cmd": "sleep 99999"
  }
}

This launches something equivalent to what we specify in the marathon app definitions above.
```